estate/models/estate_property_offer.py

```diff
@@ -15,3 +16,15 @@ class PropertyOffer(models.Model):
     )
     partner_id = fields.Many2one("res.partner", required=True)
     property_id = fields.Many2one("property", required=True)
+    validity = fields.Integer(compute="_compute_validity", inverse="_inverse_validity")
+    date_deadline = fields.Datetime()
+
+    @api.depends("date_deadline")
+    def _compute_validity(self):
+        for record in self:
+            record.validity = abs((record.date_deadline - fields.Datetime.now()).days) if record.date_deadline else 7
+
+    def _inverse_validity(self):
+        for record in self:
+            record.date_deadline = fields.Datetime.now() + timedelta(days=(record.validity))
```

Comment:
If there is no `record.validity` the date will be "today"

Is this okay?

I think we should be using something to have a default value in case `validity` is not filled